core/SConscript.firmware

```diff
@@ -127,7 +129,7 @@ if EVERYTHING:
         'ENABLE_MODULE_RECOVERY',
         'ENABLE_MODULE_ECDH',
     ]
```

Comment:
We want those removed if SECP256K1_ZKP is False as well? Just making sure

Reply:
Yes. These should affect only the compilation of stuff in `vendor/secp256k1-zkp/src/` plus we use `#ifdef SECP256K1_BUILD` in `modtrezorcrypto.c`, so this can't be defined if ZKP is not included